operators-examples/qiskit-dev-operator/operator/deploy/olm-catalog/qiskit-dev-operator/0.1.0/qiskit-dev-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "singhp11.io/v1",
+          "kind": "QiskitPlayground",
+          "metadata": {
+            "name": "example-qiskitplayground"
+          },
+          "spec": {
+            "size": 3
+          }
+        }
+      ]
+    capabilities: Basic Install
+  name: qiskit-dev-operator.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: QiskitPlayground is the Schema for the qiskitplaygrounds API
+      kind: QiskitPlayground
+      name: qiskitplaygrounds.singhp11.io
+      version: v1
+  description: Placeholder description
+  displayName: Qiskit Dev Operator
+  icon:
+  - base64data: ""
```

Comment:
you should add an icon for this, when I did the S3 one, I just created my own - you might want to do the same - same icon can be used for both or not sure if there is an icon out there already that Red Hat owns?